lib/install/config/webpacker.yml

```diff
@@ -4,6 +4,7 @@ default: &default
   source_path: app/javascript
   source_entry_path: packs
   public_output_path: packs
+  compile: false
```

Comment:
👍 Flipping the values and making the name more descriptive both sound like good changes.